conmon/cmsg.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2016 SUSE LLC
```

Comment:
As these projects (SUSE, Debian, Fedora) are not US-only, we do have to worry about international copyright law as well, and related concepts like [moral rights](https://en.wikipedia.org/wiki/Moral_rights). 

I think the Kubernetes statement is good, and would definitely prefer something like that. A similar, generic, all-encompassing statement is used for the Gradle project, and this seems to be accepted by the Debian folk: http://metadata.ftp-master.debian.org/changelogs/main/g/gradle/gradle_3.2.1-1_copyright